website/content/configuration/migration_to_crds.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Migration to CRDs
+---
+
+# Custom Resources Generator
+
+Up until release v0.12 MetalLB was configured via MetalLB's configmap. Deprecating the configmap impacts on the users' current configurations. To mitigate this, the MetalLB resource generator is an offline tool to convert the configmap into the new CRDs.
+
+## Procedure
+
+In order to use the tool, the following container must be run with the ConfigMap config.yaml mapped 
+into the /var/input folder. On success it will generate a resources.yaml file containing the new MetalLB CRD based config.
+This example assumes MetalLB was installed in the metallb-system namespace and the ConfigMap was named config:
+
+```bash
+kubectl get configmap -n metallb-system -o yaml config > config.yaml
+docker run -d -v $(pwd):/var/input quay.io/metallb/configmaptocrs
+```
+
+## Running directly against a cluster
+
+Configmaptocrs tool can also run directly against a cluster,
+and print the CRs corresponding to the configmap deployed on
+the cluster to the standard output.
+```bash
+kubectl run configmaptocrs -n metallb-system --restart=Never -it --rm --image overriden --overrides '
+{
+  "spec": {
+    "containers": [
+      {
+        "name": "configmaptocrs",
+        "image": "quay.io/metallb/configmaptocrs",
+        "command": [
+          "/configmaptocrs",
+          "-source",
+          "config",
+          "-only-data",
+          "-stdout"
+        ],
+        "volumeMounts": [
+          {
+            "name": "config",
+            "mountPath": "/var/input"
+          }
+        ]
+      }
+    ],
+    "volumes": [
+      {
+        "name": "config",
+        "configMap": {
+          "name": "config"
+        }
+      }
+    ]
+  }
+}'
+```
```

Comment:
Please provide an example of the output in this case.

Reply:
done.